libs/libcommon/src/libcommon/queue.py

```diff
@@ -198,6 +202,53 @@ def flat_info(self) -> FlatJobInfo:
         )
 
 
+class Lock(Document):
+    meta = {"collection": QUEUE_COLLECTION_LOCKS, "db_alias": QUEUE_MONGOENGINE_ALIAS}
+    key = StringField(primary_key=True)
```

Comment:
does `primary_key` imply `unique=True`? I imagine so, but could not find it in the docs of mongoengine